checked/bytes.go

```diff
@@ -54,6 +54,7 @@ func NewBytes(value []byte, opts BytesOptions) Bytes {
 		value: value,
 	}
 	b.SetFinalizer(b)
+	b.TrackObject(b.value)
```

Comment:
I can imagine one would want to trace leaks in a specific part of the code, by placing `EnableLeakDetection()` at the beginning of a section and `DisableLeakDetection()` at the end.